src/transformers/models/fuyu/image_processing_fuyu.py

```diff
@@ -186,106 +583,127 @@ def postprocess_with_tokenizer_info(
         image_placeholder_id: int,
         image_newline_id: int,
         variable_sized: bool,
-    ) -> dict:
+        patch_size: Optional[Dict[str, int]] = None,
+    ) -> FuyuBatchFeature:
         """Process images for model input. In particular, variable-sized images are handled here.
 
         Args:
-            image_input: [batch_size, subsequence_size, c, h, w] tensor of images padded to model input size.
-            image_present: [batch_size, subsequence_size] tensor of 1s and 0s indicating whether an image is present.
-            image_unpadded_h: [batch_size, subsequence_size] tensor of unpadded image heights.
-            image_unpadded_w: [batch_size, subsequence_size] tensor of unpadded image widths.
-            image_placeholder_id: The id of the image placeholder token. Comes from an associated tokenizer.
-            image_newline_id: The id of the image newline token. Comes from an associated tokenizer.
-            variable_sized: Whether to process images as variable-sized.
+            image_input (`torch.Tensor` of shape [batch_size, subsequence_size, num_channels, height, width]):
+                Tensor of images padded to model input size.
+            image_present (`torch.Tensor` of shape [batch_size, subsequence_size]):
```

Comment:
```suggestion
            image_present (`torch.Tensor` of shape [batch_size, subsequence_size, num_images]):
```

(or change the corresponding line in `get_sample_encoding`)